.jenkins/pytorch/common.sh

```diff
@@ -179,3 +179,11 @@ function get_exit_code() {
   set -e
   return $retcode
 }
+
+function file_diff_from_base() {
+  # The fetch may fail on Docker hosts, but it's not always necessary.
+  set +e
+  git fetch origin master --quiet
+  set -e
+  git diff --name-only "$(git merge-base origin master HEAD)" > "$1"
```

Comment:
Is this always right? We have stack diffs and it seems like it might be inappropriate to compare against master, as you'll pick up all the old Python changes. Perhaps it's the case that we'll only over trigger in this case? But this might not be true if there are some reverts involved??